src/test-gui/java/de/qabel/desktop/ui/accounting/identitycontextmenu/IdentityContextMenuPage.java

```diff
@@ -0,0 +1,18 @@
+package de.qabel.desktop.ui.accounting.identitycontextmenu;
+
+import com.sun.javafx.robot.FXRobot;
+import de.qabel.desktop.ui.AbstractPage;
+import org.testfx.api.FxRobot;
+
+public class IdentityContextMenuPage extends AbstractPage {
+    private IdentityContextMenuController controller;
+
+    public IdentityContextMenuPage(FXRobot baseFXRobot, FxRobot robot, IdentityContextMenuController controller) {
+        super(baseFXRobot, robot);
+        this.controller = controller;
+    }
+
+    public void edit() {
+        controller.setAlias("new alias");
```

Comment:
This is not the purpose of a GUI test. Checking that `setAlias` works could be a ControllerTest but GUI test should make sure that you can do that by interacting with the gui. This method should click on the "edit identity" menu entry, insert the new alias (that should be a method parameter) into the input dialog and confirm the change by pressing the positive button of the input dialog.